recipe/meta.yaml

```diff
@@ -197,6 +197,9 @@ outputs:
         - Rscript -e "stopifnot(capabilities('jpeg'), TRUE)"
         - Rscript -e "stopifnot(capabilities('png'), TRUE)"
         - if not exist %PREFIX%\\Lib\\R\\bin\\x64\\R.lib exit 1  # [win]
+        # Verify internal-tzcode works on macOS
+        # See https://stat.ethz.ch/pipermail/r-devel/2014-April/068745.html
+        - R -e "Sys.time()"  # [osx]
```

Comment:
Any reason not to run this test on all operating systems? It's a fast running test, and I'd think it be good to future-proof against any potential timezone related problems

Reply:
Agreed. Also switched it to error on warning in that test.